src/apps/the-exchange-app/context/SwapDataProvider.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { createContext, useContext, useState, useEffect, useMemo } from 'react';
+import { ExchangeOffer, Token } from '@etherspot/prime-sdk/dist/sdk/data';
```

Comment:
Just wondering why we're important from the dist directory directly here, I'm assuming these are types? If so they should be exported (if not then they should be, that's missing)

Reply:
Mmm... I noticed that all types from the SDK are coming from this folder